src/renderer/windows/main.js

```diff
@@ -4,8 +4,28 @@ import ReactDOM from 'react-dom';
 import injectTapEventPlugin from 'react-tap-event-plugin';
 
 import PlayerPage from '../ui/pages/PlayerPage';
+import SettingsClass from '../../main/utils/Settings';
 
 injectTapEventPlugin();
 
+// Instantiate a settings object for checking 'start minimized'
```

Comment:
This shouldn't be here, there will already be a global `Settings` object available thanks to the preload file